WordPress/Credentials/secrets-manifest.xcfilelist

```diff
@@ -1,6 +0,0 @@
-#  secrets-manifest.xcfilelist
-
-$(SRCROOT)/../.configure-files/Secrets.swift
```

Comment:
Should we keep this file and reference the files using `${HOME}/.configure/...` in order to ensure we rebuild if they change?

Reply:
> Should we keep this file

Following up to my comment [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/16806#issuecomment-876000701) I think the answer is yes. Thanks noticing this.

> reference the files using `${HOME}/.configure/...`

That would be great. Xcode has [some issues](https://github.com/mokagio/HomeFolderTest) expanding `$HOME` when used in the project file, but maybe it can manage in a build phase. I'll try 🤞

Reply:
BTW, `$HOME` in the file list file worked 🎉 See f8531a8b809d64cd294111071dd3aa863041820f